docs/competition/teams/role-descriptions/reception.md

```diff
@@ -9,23 +9,15 @@ original:
 ---
 # Reception Volunteer
 
-Reception staff control entry to the building and validate people's Media Consent Forms (MCFs) and/or wristbands. Each reception staffer has their own desk and chair.
+Reception staff control entry to the building and inform attendees that photos and videos are being taken. Each reception staffer has their own desk and chair.
 
-Here is a flow chart summarising the details below:
-
-![Reception Desk Flow](../diagrams/reception-desk-flow.svg)
-
-**Everyone** must complete a Media Consent Form, Under 16's must have this form completed by a parent or guardian. If they have signed a form, or if they bring a pre-signed form, give them a wristband and ensure you see them put it on. Also, give each competitor a competition leaflet.
-
-If they refuse, we should double check they understand what they are refusing. Media consent means that we can take photos or videos with them in and use this media to advertise ourselves. We should explain that photos and videos are being taken around the event and that it would be best for them to avoid the arena as we are also livestreaming. If they still refuse media consent provide them with a red lanyard to wear **instead** of a wristband. Instruct them they must wear this lanyard at all times around the event. Remember to give them a booklet too.
+We need to inform people that we are taking photos and videos, if they object to this we should double check they understand what they are objecting to. We are taking photos and videos that we will use to advertise ourselves. If they still object they should be given a red lanyard and told to wear them at all times, this is to ensure that we do not take photos of them. We should explain that it would be best for them to avoid the arena as we are also livestreaming.
 
 At the start of the competition day, there will be four reception staffers
 covering the entrance. Once the initial influx of people is complete, this
-will reduce. Additionally, after the initial influx, receptionists
-should allow people wearing SR wristbands to enter the building at will (they
-should check that people walking in have one, or follow the procedure above).
+will reduce.
 
-Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public should be allowed into the building if they sign a Media Consent Form. Feel free to answer, give them a booklet, and send them to Helpdesk (let them sign in) if they seem interested.
+Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public should be allowed into the building if they're happy photos and videos might be taken of them. Feel free to answer, give them a booklet, and send them to Helpdesk (let them sign in) if they seem interested.
```

Comment:
This implies we're denying entry to members of the public who don't want to be in photos -- is that so? (If so, maybe make that explicit)